app/src/main/java/com/alphawallet/app/web3/JsInjectorClient.java

```diff
@@ -203,8 +203,9 @@ private String loadInitJs(Context context) {
 
     String injectStyle(String view, String style)
     {
-        style = "<style type=\"text/css\">\n" + style + "</style>\n";
-        return style + view;
+        String injectHeader = "<head><meta name=\"viewport\" content=\"width=device-width, user-scalable=yes\" /></head>";
```

Comment:
I thought we already have something like this somewhere else in Android in order to get the width to fit (`device-width`). But anyway, on iOS, we use:

```
<meta name="viewport" content="width=device-width, initial-scale=1,  maximum-scale=1, shrink-to-fit=no">
```

The main difference is, it's effectively `user-scalable=false`. Users shouldn't be able to treat it like a browser.

Reply:
ah ok. I'll try that